scripts/local-linux/build_libcrc32c.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e -x
+
+PY_BIN="python3.7"
+
+# Check that the REPO_ROOT variable is set.
+if [[ -z "${REPO_ROOT}" ]]; then
+    echo "REPO_ROOT environment variable should be set by the caller."
+    exit 1
+fi
+
+CRC32C_INSTALL_PREFIX=${REPO_ROOT}/usr
+
+# Make sure we have an updated `pip`.
+${PY_BIN} -m pip install --upgrade pip
```

Comment:
Shouldn't need this if we're making a virtual env (and would likely need `sudo` on most machines).

Reply:
RE: `sudo`, very good point.

RE: "shouldn't need", I feel like it's "providing a service" to get the user the latest and greatest `pip`.

How would you feel if I let `PY_BIN` be user-configurable with a fallback to `python3.7`? This way users could pass do e.g.

```
PY_BIN="sudo python3.7" ./scripts/local-linux/build_libcrc32c.sh
```

Reply:
> I feel like it's "providing a service" to get the user the latest and greatest pip.

It isn't really in scope for this script.  Can you just put in a comment that suggests upgrading pip if the script fails?

Reply:
You mean like print something to `stderr` or just putting in a bash comment?

Reply:
I was thinking bash comment.

Reply:
Done.

Reply:
I meant for the comment to replace updating the host's `pip`, which I don't believe is helpful or appropriate in a script.

Reply:
Done